cli/aiconfig-editor/pages/api/aiconfig/load.ts

```diff
@@ -24,10 +29,32 @@ export default function handler(
     return res.status(500).json({ error: "No path provided" });
   }
 
-  // TODO: Needed to add node-fetch to the dependencies of aiconfig-editor for this to work
-  // This should be fixed up in library though?
   // TODO: load should probably be async?
   const aiconfig = AIConfigRuntime.load(body.path);
 
-  res.status(200).json({ aiconfig });
+  // Refine outputs for client-side rendering. We only care about displaying (and deleting)
+  // outputs directly from the editor
+  const clientAIConfig: ClientAIConfig = {
+    ...aiconfig,
+    prompts: aiconfig.prompts.map((prompt) => ({
+      ...prompt,
+      outputs: prompt.outputs?.map((output) => {
+        if (output.output_type === "execute_result") {
+          const text = aiconfig.getOutputText(prompt, output);
+          if (text) {
+            return {
+              ...output,
+              client: {
```

Comment:
The naming is a bit confusing here. I'm guessing this means this is a client-only field, but perhaps a better name may be `renderData` or something that describes what it will be used for.